test/wallet-http-test.js

```diff
@@ -278,7 +278,10 @@ describe('Wallet HTTP', function() {
     });
 
     let entered = false;
-    node.mempool.on('tx', () => {
+    node.mempool.on('tx', (tx) => {
+      if(json.hex !== tx.toHex()) {
+        return;
```

Comment:
What's this for? I think this test is designed to fail if the mempool ever emits any `'tx'` event (because we are just creating and not broadcasting).

Reply:
Actually I think we should double check this logic, looks like the event listener should be set _before_ the `createOpen()`. @tynes not sure if JS hoists these - but what we want to test is that the event is not emitted, correct?

Reply:
The problem here was that when I was running my test and tried to broadcast the TX made in advance, it failed because of this check ...

Reply:
hmmmmmmmmmm thats not good if tests interfere with each other. These tests maybe we can remove the event actually, since we test `assert.ok(!mempool.includes(json.hash));` anyway. OR if the event is really required, we need to remove the listener at the end of the test.

Reply:
So, should I remove this part of the 2 tests ?